dotnet/src/SemanticKernel.UnitTests/AI/ServiceConversionExtensionsTests.cs

```diff
@@ -440,17 +440,17 @@ public async Task AsChatClientStreamingContentConvertedAsExpected()
     [Fact]
     public async Task AsChatCompletionServiceNonStreamingContentConvertedAsExpected()
     {
-        IList<ChatMessage>? actualChatHistory = null;
+        List<ChatMessage>? actualChatHistory = null;
         ChatOptions? actualOptions = null;
 
         using IChatClient client = new TestChatClient()
         {
             CompleteAsyncDelegate = async (messages, options, cancellationToken) =>
             {
                 await Task.Yield();
-                actualChatHistory = messages;
+                actualChatHistory = messages.ToList();
                 actualOptions = options;
-                return new Microsoft.Extensions.AI.ChatResponse(new ChatMessage() { Text = "the result" });
+                return new Microsoft.Extensions.AI.ChatResponse(new ChatMessage() { Contents = [new Microsoft.Extensions.AI.TextContent("the result")] });
```

Comment:
@stephentoub note this cumbersome required way to instantiate ChatMesage.

Is the intention here to force consumer to provide a ChatRole when instantiating ChatMessage (since there's a constructor accepting a single string content, but it also requires ChatRole).

Reply:
> Is the intention here to force consumer to provide a ChatRole when instantiating ChatMessage

Yes. Is the role not known here?

Reply:
Well it's just a test, I'm assuming it's not important - will play around.